tests/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
some mess with directories. Should be `docker-compose -f docker-compose-test.yaml` in all cases

Reply:
actually this script was called from Makefile, which is outside `tests` directory. 
i've moved this script's logic to Makefile